HabTrack/app/src/main/java/com/example/habtrack/CredentialVerifier.java

```diff
@@ -32,6 +95,14 @@ public static boolean verifyPasswordFieldIsEmpty(String password) {
         }
     }
 
+    /**
+     * This method returns a {@link boolean} dependent on if the password length is considered
+     * to be valid. It also updates passwordVerifyFailureMessage if the password is of an invalid
+     * length. Method parameter password is expected to be non null.
+     *
+     * @param password  Takes input of userName of type {@link String}.
+     * @return          The return type is {@link boolean}.
+     */
     public static boolean verifyPasswordFieldLength(String password) {
         if (password.length() < 8) {
```

Comment:
I realize that instead of using "magic" numbers we can declare a constant `MIN_PASSWORD_LENGTH` and use that

Reply:
Not exactly sure where to declare this, I'm assuming as a global variable outside the class? I will leave this unchanged for someone else to update.